### ISSUE TITLE
AR application based on real world location

### ISSUE THREAD

**Issue by liqiangdada** [SceneView/sceneform-android#329]
I want to place some models according to longitude and latitude. How to develop them?

**Comment by ThomasGorisse** [comment]
https://github.com/appoly/ARCore-Location
Could you create a PR with it?

**Comment by liqiangdada** [comment]
I noticed him. It seems that he stopped maintenance in 2018. I ran his demo and the effect was too bad! The mark is always moving, not stationary!

**Comment by kustraslawomir** [comment]
> 

It's moving because your smartphone updates your position and your azimuth. So the markers are changing their position. 
This behavior is quite normal. You could try set only once the LatLng position on the arScene, maybe it will be more stable, but it won't fix the position if the marker has been rendered in the wrong place at first run & the user will move.

**Comment by liqiangdada** [comment]
> > 
> 
> It's moving because your smartphone updates your position and your azimuth. So the markers are changing their position. This behavior is quite normal. You could try to set only once the LatLng position on the arScene, maybe it will be more stable, but it won't fix the position if the marker has been rendered in the wrong place at the first run & the user will move.
> 
> I personally use the library from appoly, there is no better opensource alternative for now I think.

Arkit doesn't seem to have an Android version What library do you use?
thanks.

**Comment by MrCsabaToth** [comment]
Latest 2022 Google I/O news: Geospatial API is able to do exactly that. Soon I'll try to use sceneform and that API together https://github.com/google-ar/codelab-geospatial

**Comment by MrCsabaToth** [comment]
It is a hard task to determine orientation and location.
